storage/client_range_gc_test.go

```diff
@@ -41,6 +41,20 @@ func TestRangeGCQueueDropReplica(t *testing.T) {
 	// Increment the clock's timestamp to expire the leader lease.
 	mtc.manualClock.Increment(int64(storage.DefaultLeaderLeaseDuration) + 1)
 
+	// Make sure the range is not yet removed from the store.
+	numTrials := 10
+	for i := 0; i < numTrials; i++ {
+		store := mtc.stores[1]
+		store.ForceRangeGCScan(t)
+		if _, err := store.GetRange(raftID); err != nil {
+			t.Error("unexpected range removal")
+		}
+		time.Sleep(10 * time.Millisecond)
```

Comment:
Do we need to really sleep here, or can we just increment the manual clock? If the real sleep matters, let's reduce numTrials to 3 so we don't have to wait 100ms for nothing to happen.